src/app/(explore)/[...slug]/page.tsx

```diff
@@ -15,30 +15,15 @@ import Image from "next/image";
 export const dynamicParams = false;
 
 export function generateStaticParams() {
-  // The flattenedPath and SlugParams are destuctured from the value of the currentElement
-  const all = allTranscripts.reduce((acc, transcript) => {
-    // params and slugAsParams are more or less the same, continued using both from previous code
-    const params = transcript._raw.flattenedPath.split("/");
-
-    // transcript.slugAsParams is typed as a list from contentLayer but its manually casted to `string[]` to specify it's exact type
-    const slugAsParams = transcript.slugAsParams as unknown as string[];
-
-    const is_non_english_dir_or_transcript = /\w+\.[a-z]{2}\b/.test(params[params.length - 1]);
-
-    const lastRouteIndex = params[params.length - 1].includes("index");
-
-    if (!lastRouteIndex) {
-      acc.push({ slug: slugAsParams });
-      // converts slug ['adopting-bitcoin', '_index'] to ['adopting-bitcoin']
-      // skips non english dir e.g ['adopting-bitcoin', '_index.es'] is skipped
-    } else if (!is_non_english_dir_or_transcript) {
-      acc.push({ slug: slugAsParams.slice(0, slugAsParams.length - 1) });
+  const allSlugs = allContentSources.map(({ slugAsParams, language }) => {
+    if (language === "en") {
+      return { slug: slugAsParams };
+    } else {
```

Comment:
in `contentlayer.config.ts`, non english language transcripts were skipped.
Let's skip languages here as well and tackle it in a separate PR.

handling languages require some additional changes to the content tree and that is out of scope for this PR